docs/docs/migration/migration-from-v5-to-v6.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Migrating from v5 to v6
+sidebar_position: 1
+---
+
+# Migrating from v5 to v6
+
+First, execute `npx @comet/upgrade@latest v6` in the root of your project. 
+It automatically installs the new versions of all `@comet` libraries and handles some of the necessary renames.
+
+<details>
+
+<summary>Renames handled by @comet/upgrade</summary>
+
+- `JobStatus` -> `KubernetesJobStatus` in API
+- `@SubjectEntity` -> `@AffectedEntity` in API
+- `BuildRuntime` -> `JobRuntime` in Admin
+
+</details>
```

Comment:
I moved the handled renames to an accordion (following @nsams comment https://github.com/vivid-planet/comet/pull/1637#issuecomment-1916735125)